External/FEXCore/Source/Interface/Core/Interpreter/InterpreterOps.cpp

```diff
@@ -397,13 +398,19 @@ Res GetSrc(void* SSAData, IR::OrderedNodeWrapper Src) {
 [[noreturn]]
 static void StopThread(FEXCore::Core::InternalThreadState *Thread) {
   Thread->CTX->StopThread(Thread);
-  std::unexpected();
+
+  // If the StopThread signal is delayed for any reason, spin forever
+  for (;;)
```

Comment:
How about `LogMan::Msg::A` here?

Reply:
I'm really not sure that tgkill has any kind of synchronicity guarantee for doing a self kill. 

It might be somewhat common to reach here. 

I'm working on another PR which just stops the thread synchronously.

Reply:
tgkill to itself will immediately switch the thread to the signal handler. It's abused in the Linux syscall handler for pending signal handling.

Reply:
Hmm... I checked the linux source code and couldn't really see a code path which guaranteed that.